README.md

```diff
@@ -160,10 +160,13 @@ By default the size of the container (ram, disk, cpu shares) uses the package in
 the internal "sdc_" set of packages closest to 'ram=1024 MiB'. The "sdc_"
 packages are really only applicable for development. More appropriate for
 production is a set of packages separate from "sdc_". The following can be
-run to add a number of "t4-*" packages and to configure the Docker service
+run from the global zone to add a number of "t4-*" packages and to configure the Docker service
 to use them:
 
-    sdc-login docker /opt/smartdc/docker/tools/gen_packages.js
+
+    papi_svc=$(sdc-sapi /services?name=papi | json -Ha uuid)
+    sdc-sapi /services/$papi_svc -X PUT -d '{ "metadata": { "IGNORE_CPU_CAP": true } }'
+    ./tools/gen_packages.js
```

Comment:
Yes, after talking to JoshW, it appears that you need to run gen_packages
in the global zone - not within docker's zone. Also, I was unclear about
the base path of the directory used in previous commands in the
documentation (specified as ./tools), so I assumed that it was just a copy
of the sdc-docker repo. That's why I replaced the path with a relative
path. Also, when I would attempt to run the "sapiadm" command, it would
fail and I would have to set IGNORE_CPU_CAP in order for it to work.

All of this was on the latest version of the platform images and libraries.

On Wed, Jun 10, 2015 at 12:52 PM, Trent Mick <notifications@github.com>
wrote:

> In README.md
> <https://github.com/joyent/sdc-docker/pull/38#discussion_r32158829>:
>
> >  to use them:
> >
> > -    sdc-login docker /opt/smartdc/docker/tools/gen_packages.js
> > +
> > +    papi_svc=$(sdc-sapi /services?name=papi | json -Ha uuid)
> > +    sdc-sapi /services/$papi_svc -X PUT -d '{ "metadata": { "IGNORE_CPU_CAP": true } }'
> > +    ./tools/gen_packages.js
>
> Elijah,
>
> How did the old way fail? There is no "tools/gen_packages.js" in the GZ so
> I don't see how to run your new instructions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joyent/sdc-docker/pull/38/files#r32158829>.
>



-- 
-Elijah